### DISCUSSION TITLE
Best way to to have an input indicate it is a dollar amount?

### DISCUSSION THREAD

**Discussion by tomvonheill** [cloudscape-design/components#953] [Q&A] [answered]
I have a field that is being input as a dollar amount, and am wondering the best way to show the user this? Any one have a best practice.

I was thinking about just appending a dollar sign to the front of the input but that doesn't feel right. Should it just be a hint?

Looking for some strong opinions (i'm a backend guy looking to not destroy this these UI standards)

<img width="788" alt="image" src="https://user-images.githubusercontent.com/10145413/231778457-50ccb7c1-25b2-407e-b4e2-41bbc2af3e63.png">

**Comment by katiegeorge** [comment, marked as answer]
Good question! There are a couple different ways you could do this that would all be good industry practice, so it's up to you which one you ultimately end up using.

1. Add an indicator directly to the field label

<img width="631" alt="image" src="https://github.com/cloudscape-design/components/assets/9701338/7e6e762c-93fc-42ee-930c-242f3943895b">

2. Add an indicator inside the constraint text.
<img width="632" alt="image" src="https://github.com/cloudscape-design/components/assets/9701338/05834979-ef83-489f-9ce6-f7ef78258b15">

3. Add an indicator inline, as a prefix or suffix.
<img width="227" alt="image" src="https://github.com/cloudscape-design/components/assets/9701338/929c579c-01fa-41eb-8a93-1b9290e79a7d">
<img width="246" alt="image" src="https://github.com/cloudscape-design/components/assets/9701338/ecf4af6b-386c-4ef2-8795-aef7057237e5">

One small aspect to keep in mind with adding a prefix here is that if you have several fields lined up vertically, the alignment could look off.

Remember to add `type="number"` to the Input as an added affordance so that the user can't accidentally type a letter value!
